src/ext/NetFx/ca/precomp.h

```diff
@@ -2,6 +2,8 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
 
 
+#include <string>
```

Comment:
Was this necessary?

Reply:
At one point during the development yes, but not anymore. I'll revert it.